site-packages/integralstor/urbackup.py

```diff
@@ -90,42 +87,29 @@ def get_log_data(logids=None):
     else:
         return log_data, None
 
-
 def get_logs(date_on=None, date_from=None, date_till=None):
-    logs = None
+    logs = []
     qry = ''
+    client_id_list = []
     try:
         urb_db, err = config.get_urbackup_db_path()
         if err:
             raise Exception(err)
         if not urb_db:
             raise Exception("Path not set")
 
-        if date_on is not None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) == "%s"' % (
-                date_on)
-        elif date_from is not None and date_till is not None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) between "%s" and "%s"' % (
-                date_from, date_till)
-        elif date_from == None and date_till == None:
-            qry = 'select * from logs'
-        elif date_from and date_till == None:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) >= "%s"' % (
-                date_from)
-        elif date_from == None and date_till:
-            qry = 'select * from logs where strftime("%%Y-%%m-%%d",created) <= "%s"' % (
-                date_till)
-        logs, err = db.get_multiple_rows(urb_db, qry)
-        if err:
-            raise Exception(err)
+        qry = 'select id from clients'
+        client_id = db.get_multiple_rows(urb_db, qry)
 
-        """
-        # It's not efficient to fetch all log data
-        log_data, err = get_log_data()
-        if log_data:
-            for log in logs:
-                log['logdata'] = [data['data'] for data in log_data if data['logid'] == log['id']]
-        """
+        for i in client_id[0]:
+                client_id_list.append(i['id'])
+
+        for j in client_id_list:
+                qry = 'select * from logs where clientid = %d order by id desc limit 1' % (j)
+                k = db.get_multiple_rows(urb_db, qry)
```

Comment:
Instead of variables like 'k', it is better to use more descriptive variable names